app/routes/circulars.events.$slug.tsx

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { LoaderFunctionArgs } from '@remix-run/node'
+import { Link, useLoaderData } from '@remix-run/react'
+import { Icon } from '@trussworks/react-uswds'
+import invariant from 'tiny-invariant'
+
+import type { Synonym } from './synonyms/synonyms.lib'
+import {
+  getAllSynonymMembers,
+  getSynonymsBySlug,
+} from './synonyms/synonyms.server'
+import { ToolbarButtonGroup } from '~/components/ToolbarButtonGroup'
+import { PlainTextBody } from '~/components/circularDisplay/Body'
+import { FrontMatter } from '~/components/circularDisplay/FrontMatter'
+import { feature } from '~/lib/env.server'
+import type { BreadcrumbHandle } from '~/root/Title'
+
+export const handle: BreadcrumbHandle = {
+  breadcrumb: 'Circular Event Group',
+}
+
+export async function loader({
+  params: { slug },
+  request: { url },
+}: LoaderFunctionArgs) {
+  if (!feature('SYNONYMS')) throw new Response(null, { status: 404 })
+  invariant(slug)
+
+  const { searchParams } = new URL(url)
+  const view = searchParams.get('view') || 'group'
+  const limit = searchParams.get('limit') || '20'
+  const page = searchParams.get('page') || '1'
```

Comment:
There's no need to get these in the loader; they are available on the client side.